src/service/viewport/viewport-impl.js

```diff
@@ -403,9 +405,15 @@ export class ViewportImpl {
 
   /** @override */
   scrollIntoView(element) {
-    return this.getScrollingContainerFor_(element).then((parent) =>
-      this.scrollIntoViewInternal_(element, parent)
-    );
+    if (IS_SXG) {
+      const {promise} = new Deferred();
```

Comment:
What does this do? It looks like it just returns a never-resolving promise? Did you mean to just return `return Promise.resolve()`?

Reply:
Yes, it should return `resolve()`. I've fixed this just now.

Reply:
I think you forgot to push the commits then. I'm still seeing the broken code.

Reply:
Sorry about that. I pushed it just now.